egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -298,8 +313,9 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
   extra_opts+=(--transform-dir $transform_dir)
   extra_opts+=(--left-context $left_context)
   extra_opts+=(--right-context $right_context)
-  extra_opts+=(--valid-left-context $((chunk_width + left_context)))
-  extra_opts+=(--valid-right-context $((chunk_width + right_context)))
+  extra_opts+=(--valid-left-context $((minibatch_chunk_size + left_context)))
```

Comment:
Keep this as chunk-width. I think that might lead to a better match with training condition.

@danpovey  Do you agree ?